src/workerd/api/sql.h

```diff
@@ -107,6 +107,19 @@ class SqlStorage final: public jsg::Object, private SqliteDatabase::Regulator {
       return pageSize.emplace(db.run("PRAGMA page_size;").getInt64(0));
     }
   }
+
+  // Utility functions to convert SqlValue, SqlRow, and Array<SqlValue> to JS values. In some
+  // cases we end up having to do this conversion before actually returning, so we can't have
+  // JSG do it. We can't use jsg::TypeHandler because SqlValue contains StringPtr, which doesn't
+  // support unwrapping. We don't actually ever use unwrapping, but requesting a TypeHandler forces
+  // JSG to try to generate the code for unwrapping, leading to compiler errors.
+  //
+  // TODO(cleanup): Think hard about how to make JSG support this better. Part of the problem is
+  //   that we're being too clever with optimizations to avoid copying strings when we don't need
+  //   to.
+  static jsg::JsValue wrapSqlValue(jsg::Lock& js, SqlValue value);
+  static jsg::JsObject wrapSqlRow(jsg::Lock& js, SqlRow row);
+  static jsg::JsArray wrapSqlRowRaw(jsg::Lock& js, kj::Array<SqlValue> row);
```

Comment:
I don't think this function is used anywhere.

Reply:
It isn't now, but likely to be in the future when we figure out `raw().toArray()` vs. `toRawArray()` or whatever.

Reply:
I'd probably remove it, but I know we're on a schedule here.